assets/models/system/tiiuae-falcon-7b/description.md

```diff
@@ -0,0 +1,124 @@
+
+#### Speeds, Sizes, Times
+
+Training happened in early March 2023 and took about two weeks. 
+
+
+#### Evaluation
+
+*Paper coming soon*.
+
+See the [OpenLLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) for early results.
+
+
+#### Technical Specifications 
+
+#### Model Architecture and Objective
+
+
+Falcon-7B is a causal decoder-only model trained on a causal language modeling task (i.e., predict the next token).
+
+The architecture is broadly adapted from the GPT-3 paper ([Brown et al., 2020](https://arxiv.org/abs/2005.14165)), with the following differences:
+
+* **Positionnal embeddings:** rotary ([Su et al., 2021](https://arxiv.org/abs/2104.09864));
+* **Attention:** multiquery ([Shazeer et al., 2019](https://arxiv.org/abs/1911.02150)) and FlashAttention ([Dao et al., 2022](https://arxiv.org/abs/2205.14135));
+* **Decoder-block:** parallel attention/MLP with a single layer norm.
+
+
+| **Hyperparameter** | **Value** | **Comment**                            |
+|--------------------|-----------|----------------------------------------|
+| Layers             | 32        |                                        |
+| `d_model`          | 4544      | Increased to compensate for multiquery                                       |
+| `head_dim`         | 64        | Reduced to optimise for FlashAttention |
+| Vocabulary         | 65024     |                                        |
+| Sequence length    | 2048      |                                        |
+
+
+#### Compute Infrastructure
+
+#### Hardware
+
+Falcon-7B was trained on AWS SageMaker, on 384 A100 40GB GPUs in P4d instances. 
+
+#### Software
+
+Falcon-7B was trained a custom distributed training codebase, Gigatron. It uses a 3D parallelism approach combined with ZeRO and high-performance Triton kernels (FlashAttention, etc.)
+
+#### License
+
+Falcon-7B is made available under the Apache 2.0 license.
+
+# Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint-dolly.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon
+
+### Model Evaluation Sample
+
+Task| Use case| Dataset| Python sample (Notebook)| CLI with YAML
+|--|--|--|--|--|
+Text generation | Text generation | <a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank"> cnn_dailymail </a> | <a href="https://aka.ms/azureml-eval-sdk-text-generation/" target="_blank">evaluate-model-text-generation.ipynb</a> | <a href="https://aka.ms/azureml-eval-cli-text-generation/" target="_blank">evaluate-model-text-generation.yml</a>
+
```

Comment:
Could you please add FT samples as well?

Reply:
Done